dist/ovh-api-services.js

```diff
@@ -5810,12 +5810,42 @@ angular.module("ovh-api-services").service("OvhApiMeSshKey", ["$injector", funct
     };
 }]);
 
-angular.module("ovh-api-services").service("OvhApiMeSshKeyLexi", ["$injector", "$resource", function ($injector, $resource) {
+angular.module("ovh-api-services").service("OvhApiMeSshKeyLexi", ["$resource", "$cacheFactory", function ($resource, $cacheFactory) {
     "use strict";
 
-    var req = $resource("/api/me/sshKey");
+    var cache = $cacheFactory("OvhApiMeSshKeyLexi");
+    var queryCache = $cacheFactory("OvhApiMeSshKeyLexiQuery");
 
-    return req;
+    var interceptor = {
+        response: function (response) {
+            cache.remove(response.config.url);
+            queryCache.removeAll();
+            return response.resource;
+        }
+    };
+
+    var resource = $resource("/me/sshKey/:keyName", { keyName: "@keyName" }, {
+        query: { method: "GET", cache: queryCache, isArray: true },
+        get: { method: "GET", cache: cache },
+        create: { method: "POSt", interceptor: interceptor },
+        edit: { method: "PUT", interceptor: interceptor },
+        "delete": { method: "DELETE", interceptor: interceptor }
```

Comment:
-- double quote ?

It's not a keyword here, it's a JSON ^.^

Reply:
It is a keyword. All the delete in ovh-api-services have double quotes.